qa/L0_infer/install_and_test.sh

```diff
@@ -29,8 +29,10 @@
 # dependencies to run L0_infer tests
 apt-get update && \
     apt-get install -y --no-install-recommends \
+         curl \
+         jq \
          python3 \
-         python3-pip && \
+         python3-pip
```

Comment:
Why break these compound statements?

Reply:
It doesn't have to be connected. @GuanLuo suggested I remove it